hardware/src/base-station-sketch2/sketch_aug16a.ino

```diff
@@ -0,0 +1,55 @@
+#include "CmdMessenger.h"
+
+/* Define available CmdMessenger commands */
+enum {
+    current_status,
+    status_is,
+    think_for_me,
+    thought_is,
+    error,
+};
+
+/* Initialize CmdMessenger -- this should match PyCmdMessenger instance */
+const int BAUD_RATE = 9600;
+CmdMessenger c = CmdMessenger(Serial,',',';','/');
```

Comment:
I agree, I am not even sure what they're doing to be fair. That was just copied from an example